example/src/main/java/com/stripe/example/activity/PaymentIntentActivity.java

```diff
@@ -0,0 +1,255 @@
+package com.stripe.example.activity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+import com.stripe.android.PaymentConfiguration;
+import com.stripe.android.Stripe;
+import com.stripe.android.model.Card;
+import com.stripe.android.model.PaymentIntent;
+import com.stripe.android.model.PaymentIntentParams;
+import com.stripe.android.model.SourceParams;
+import com.stripe.android.view.CardInputWidget;
+import com.stripe.example.R;
+import com.stripe.example.controller.ErrorDialogHandler;
+import com.stripe.example.controller.ProgressDialogController;
+import com.stripe.example.module.RetrofitFactory;
+import com.stripe.example.service.StripeService;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.Callable;
+
+import okhttp3.ResponseBody;
+import retrofit2.Retrofit;
+import rx.Observable;
+import rx.Subscription;
+import rx.android.schedulers.AndroidSchedulers;
+import rx.functions.Action0;
+import rx.functions.Action1;
+import rx.schedulers.Schedulers;
+import rx.subscriptions.CompositeSubscription;
+
+public class PaymentIntentActivity extends AppCompatActivity {
+    private static final String TAG = PaymentIntentActivity.class.getName();
+
+    private static final String RETURN_SCHEMA = "stripe://payment_intent_return";
```

Comment:
This should be `RETURN_URL` (or similar) - the `RedirectActivity` had the url split into multiple constants, one of which was the schema, and two different hosts.